mahotas/tests/LineDetectorByHough.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2008-2014, Jasson <1917098992@qq.com>
+# License: MIT (see COPYING file)
+
+import math
+import numpy as np
+
+class Line():
```

Comment:
Could this be replaced with a `namedtuple`?

Reply:
Yes, I read the API, and replaced with a namedtuple, it do better!